instrumentation/apache-log4j-2/src/main/java/com/nr/agent/instrumentation/log4j2/AgentUtil.java

```diff
@@ -44,7 +44,7 @@ public static void recordNewRelicLogEvent(LogEvent event) {
             Throwable throwable = event.getThrown();
 
             if (shouldCreateLogEvent(message, throwable)) {
-                ReadOnlyStringMap contextData = event.getContextData();
+                Map<String, String> contextData = event.getContextMap();
```

Comment:
All implementations of this method (in 2.17) simply call `getContextDate().toMap()`.